.gitignore

```diff
@@ -6,3 +6,4 @@ results/
 testing/
 testing*
 *.pyc
+null/
```

Comment:
What is this folder?

Reply:
A `null` folder sometimes generated from a failed analysis, so git will ignore checking o this folder.